### PR TITLE
Feat: add map based in user geolocation

### PR DESCRIPTION
**Pull request by gustavosf2003**
In this PR it was added the functionality that shows a map based in user geolocation .The following packages were used:

[react-native-geocoding](https://www.npmjs.com/package/react-native-geocoding): a package that retrieves coordinates from a string address.
[react-native-maps](https://www.npmjs.com/package/react-native-maps?activeTab=readme): a package that displays a Google Maps map  based on coordinates
[react-native-dotenv](https://www.npmjs.com/package/react-native-dotenv): a package that allows the use of .env files in the app

Please note that the current address is hardcoded, as the dynamic address based on the user's location has not yet been implemented.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,6 +9,7 @@ npm-debug.*
 *.mobileprovision
 *.orig.*
 web-build/
+.env
```
Comment (kkfuri) [inline .gitignore:12]: seria bom deixar um .env.example ou atualizar o readme

File: src/components/MapComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            Failed to load map
+          </Text>
+        </View>
+      </View>
+    );
+  }
+
+  return (
+    <MapView
+      style={{ height: 200, width: "100%", borderRadius: 12, marginTop: 12 }}
```
Comment (kkfuri) [inline src/components/MapComponent.tsx:43]: teria que ser 100% screen height, não?

File: types/env.d.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+declare module "@env" {
```
Comment (kkfuri) [inline types/env.d.ts:1]: isso ja resolve a falta de .env.example

**Review by kkfuri — APPROVED** [review]

**Review by vitor-afonso — APPROVED** [review]

File: src/components/MapComponent.tsx (hunk 2 above)
Comment (gustavosf2003) [reply src/components/MapComponent.tsx:43]: Faz sentido, ai altero o tamanho no componente pai

**Pull request merged**